themes/bootstrap5/templates/header.phtml

```diff
@@ -0,0 +1,117 @@
+<?php $account = $this->auth()->getManager(); ?>
+<div class="banner container navbar">
+  <a class="navbar-brand lang-<?=$this->layout()->userLang ?>" href="<?=$this->url('home')?>">VuFind</a>
+  <button type="button" class="navbar-toggle" data-bs-toggle="collapse" data-bs-target="#header-collapse" aria-label="<?=$this->transEscAttr('mobile_toggle_navigation_text')?>">
+    <?=$this->icon('ui-menu') ?>
+  </button>
+  <?php if (!isset($this->layout()->renderingError)): ?>
+    <div class="collapse navbar-collapse" id="header-collapse">
+      <nav>
+        <ul class="nav navbar-nav navbar-right flip">
+          <?php if ($this->feedback()->tabEnabled()): ?>
+            <li>
+              <a class="icon-link" id="feedbackLink" data-lightbox href="<?=$this->url('feedback-home') ?>">
+                <?=$this->icon('feedback', 'icon-link__icon') ?>
+                <span class="icon-link__label"><?=$this->transEsc('Feedback')?></span>
+              </a>
+            </li>
+          <?php endif; ?>
+          <?php $cart = $this->cart(); ?>
+          <?php if ($cart->isActive()): ?>
+            <li id="cartSummary" aria-live="polite" aria-atomic="true">
+              <a id="cartItems" class="icon-link" data-lightbox title="<?=$this->transEscAttr('View Book Bag')?>" href="<?=$this->url('cart-home')?>">
+                <?=$this->icon('cart', 'icon-link__icon') ?>
+                <span class="icon-link__label">
+                  <span class="sr-only"><?=$this->transEsc('Book Bag')?>:</span>
+                  <span role="status"><strong><?=count($cart->getItems())?></strong> <?=$this->transEsc('items')?></span>
+                  <span class="full<?=!$cart->isFull() ? ' hidden' : '' ?>">(<?=$this->transEsc('bookbag_full') ?>)</span>
+                </span>
+              </a>
+            </li>
+          <?php endif; ?>
+          <?php if (is_object($account) && $account->loginEnabled()): // hide login/logout if unavailable ?>
+            <?php if ($account->getIdentity()): ?>
+              <li class="logoutOptions<?php if ($account->dropdownEnabled()): ?> with-dropdown<?php endif ?>">
+                <a href="<?=$this->url('myresearch-home', [], ['query' => ['redirect' => 0]])?>" class="icon-link">
+                  <span id="account-icon" class="icon-link__icon" role="link"><?=$this->icon('my-account') ?></span>
```

Comment:
I'm unsure about the addition of `role="link"` here. Since we're inside a link already, is it superfluous?

Reply:
Sort of, yes, but Bootstrap's tooltip component adds, for better or worse, aria-label attribute to the span, and that's not allowed without a proper role.

Reply:
I see! Thanks for explaining!